xyz

```diff
@@ -167,4 +173,4 @@ run "git commit$([[ $edit == true ]] && printf ' --edit') --message '$message'"
 run "git tag --annotate '$tag' --message '$message'"
 run "git push --atomic '$repo' 'refs/heads/$branch' 'refs/tags/$tag'"
 
-node -e 'process.exit(require("./package.json").private === true ? 1 : 0)' && run "npm publish"
+node -e 'process.exit(require("./package.json").private === true ? 1 : 0)' && run "$publish_command"
```

Comment:
Should we be concerned with quoting at all? Do all of the following work?

  - `--publish-command 'echo foo'`
  - `--publish-command 'echo "bar"'`
  - `--publish-command "echo 'baz'"`